app/controllers/organizations_controller.rb

```diff
@@ -1,4 +1,19 @@
 # frozen_string_literal: true
 
 class OrganizationsController < ApplicationController
+  def show
+    @organization = Organization.find(params[:id])
+    render json: json_structure(@organization)
```

Comment:
Deberías modificar el nombre del método por `public` y deberías implementar una serialización de los datos usando la gema `jbuilder`, que ya está instalada en el proyecto. Te paso el link a los [docs de jbuilder](https://github.com/rails/jbuilder) y un pequeño [ejemplo de implementación](https://www.bigbinary.com/learn-rubyonrails-book/use-jbuilder-to-render-json)

Reply:
Muchas gracias, cambios realizados